src/cosmic_ray/tools/html.py

```diff
@@ -89,7 +89,8 @@ def _generate_html_report(db, only_completed, skip_success):
                                         text('Complete: {} ({:.2f}%)'.format(
                                             num_complete, num_complete / num_items * 100))
                                     with tag('p'):
-                                        text('Survival rate: {:.2f}%'.format(survival_rate(db)))
+                                        num_killed = kills_count(db)
+                                        text('Survived mutants: {} ({:.2f}%)'.format(num_complete - num_killed, survival_rate(db)))
```

Comment:
Change this to "Surviving mutants".

Reply:
Addressed in both `html.py` and `report.py`